typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "<%= moduleName %>",
+	"version": "0.0.0",
+	"description": "",
+	"license": "MIT",
+	"repository": "<%= githubUsername %>/<%= moduleName %>",
+	"author": {
+		"name": "<%= name %>",
+		"email": "<%= email %>",
+		"url": "<%= website %>"
+	},
+	"engines": {
+		"node": ">=6"
+	},
+	"scripts": {
+		"build": "del dist && tsc",
+		"prepare": "npm run build",
+		"pretest": "npm run build",
```

Comment:
This is not needed with `ts-node`.

Reply:
While developing a package of mine I added this as the error messages where more helpful than the shortened version Ava displayed.
Also there error was there way faster.

But you are right, in a perfect "end-scenario" this shouldn't be there.